Bottom: cd2bee162418bbb57796de01c9211dc4a4977a09
Top:    6e1296a5a6377bebe8ad92cca313d0dd32170f35
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-15 10:55:41 -0700

Refresh of inspect

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp b/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp
index 70654a42384..181a2e84083 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp
@@ -652,10 +652,8 @@ bool ScriptInterpreterPythonImpl::SetStdHandle(FileSP file_sp,
   PythonDictionary &sys_module_dict = GetSysModuleDictionary();
 
   auto new_file = PythonFile::FromFile(file, mode);
-  if (!new_file) {
-    llvm::consumeError(new_file.takeError());
+  if (!new_file)
     return false;
-  }
 
   save_file = sys_module_dict.GetItemForKey(PythonString(py_name));
